src/Helpers/ConfigHelper.php

```diff
@@ -113,7 +113,7 @@ public function getImportOrderSince() {
   /**
    * @return string
    */
-  public function getPluginVersion() {
+  public function getPluginVersion(): string {
```

Comment:
Double-check that the other use(s) of this method are okay with us adding the type restriction here.